substrate/dex/pallet/src/tests.rs

```diff
@@ -18,7 +18,10 @@
 // It has been forked into a crate distributed under the AGPL 3.0.
 // Please check the current distribution for up-to-date copyright and licensing information.
 
-use crate::{mock::*, *};
+use crate::{
+  mock::{*, MEDIAN_PRICE_WINDOW_LENGTH},
```

Comment:
Because we have 2 of these, One in the `primitives/src/constants.rs` and other one is in the `dex/pallet/src/mock.rs`, compiler doesn't know which to use if we do `*` on both files. So we import the `mock.rs` one explicityly.

Reply:
Ugh. That's a mess. Probably better to remove a glob import (as they introduce such issues) but it's fine.